client/landing/jetpack-cloud/components/sidebar/style.scss

```diff
@@ -0,0 +1,48 @@
+// Menu links
```

Comment:
As much as I like how close this is to the original designs, I'd like @Automattic/jetpack-design to say what they prefer here. Should we be styling everything as we do here, or should we be using what Calypso provides us for starters?

Reply:
I personally really love how with just a few styles @delawski gave the sidebar a new look. Okay, it's not _that_ different, but the added tiny bit of customization really drives this home for me.

So, going forward, I'd say we could add a couple of styles if the element really needs it. As mentioned yesterday, the entire UI for Scan should be pretty much a 1:1 reuse of existing components and I imagine Backup will follow suit, so when styles could be needed it's not an entire rewrite of each component.

I'd like to know your thoughts too as this possibly impacts your work more than ours, though I'll say I'm more than happy to jump on _any_ PR and add custom styles when they're needed.